.github/workflows/main-lab.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         env:
           CI: true
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-          NPM_TOKEN: ${{secrets.NPM_TOKEN}}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Are these already added in DHI/react-components, or someone with owner permissions has to add them?

Reply:
The latter I believe. I think that if we are owners of that repo, we can use our personal OAuth